lib/components/Datepicker/Datepicker.tsx

```diff
@@ -85,19 +90,19 @@ export const Datepicker = React.forwardRef<HTMLInputElement, DatepickerProps>(fu
             }}
             onBlur={onBlur}
           />
-          <ArkDatePicker.Trigger className="material-symbols-outlined select-none rounded-r-[10px] border-y-[5px] border-r-[5px] border-border-gray bg-white p-[11px] text-secondary-gray">
+          <ArkDatePicker.Trigger className="material-symbols-outlined select-none rounded-r border-y border-r border-border-gray bg-white p-5 text-secondary-gray">
             calendar_month
```

Comment:
Datepicker's input has no calendar icon, but would be annoying without the button OR do not want to force it to open when input gains focus.